### PR TITLE
Added methods to allow mutually exclusive

### PR DESCRIPTION
**Pull request by SamGodwin2**
### What is the context of this PR?
This PR add the methods allowing Publisher to convert Author mutually exclusive answers to Runner acceptable format. 

### How to review 
Tests are passing and the new schema pass validation and is displayed correctly within Runner.

### Dependency 
https://github.com/ONSdigital/eq-author-api/pull/109

### CODE REVIEW

**Comment by SamGodwin2** [comment]
[Mutually exclusive answer options - publisher](https://trello.com/c/xXnpgPTE/575-mutually-exclusive-answer-options-publisher)

File: src/eq_schema/Answer.js [hunk 1]
```diff
@@ -44,6 +44,13 @@ class Answer {
       this.options = answer.options.map(this.buildOption);
     }
 
+    if (!isNil(answer.mutuallyExclusiveOption)) {
+      this.type = "MutuallyExclusiveCheckbox";
+      this.options = this.options.concat(
+        this.buildOption(answer.mutuallyExclusiveOption)
+      );
+    }
+
```
Comment (samiwel) [inline src/eq_schema/Answer.js]: Needs to be moved below `other` option logic.

File: src/eq_schema/Answer.test.js [hunk 2]
```diff
@@ -135,6 +135,51 @@ describe("Answer", () => {
...
+          description: "Another description"
+        },
+        {
+          label: "Option three",
+          value: "Option three",
+          description: "I am mutually Exclusive"
+        }
+      ]);
+    });
+
```
Comment (samiwel) [inline src/eq_schema/Answer.test.js]: Probably worth adding a test with mutually exclusive option and an `other` option to ensure that other appears before the mutually exclusive option.

**Comment by samiwel** [comment]
Needs rebasing and squashing 👍

**Pull request merged**